subsys/suit/ipuc/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Module giving an access to in-place updateable components
+
+zephyr_interface_library_named(suit_ipuc)
+target_include_directories(suit_ipuc INTERFACE include)
```

Comment:
Consider adding the following lines (due to direct includes from the interface header):
```
target_link_libraries(suit_ipuc INTERFACE suit_platform_err)
target_link_libraries(suit_ipuc INTERFACE suit_metadata)
```